.github/workflows/auto_deployment_on_label.yml

```
@@ -0,0 +1,71 @@
+          token: ${{ secrets.GITHUB_TOKEN }}
+          env: ${{ github.head_ref }}
+
+  trigger-autodeployment-on-label:
+    runs-on: ubuntu-22.04
+    if: ${{ github.event.label.name == 'autodeploy' }}
+    permissions:
+      deployments: write
+    steps:
+      - name: Check for in-progress CI build on this branch
+        id: check_build
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)